src/rdf4cpp/rdf/datatypes/registry/LiteralDatatypeImpl.hpp

```diff
@@ -13,20 +13,23 @@
 
 namespace rdf4cpp::rdf::datatypes::registry {
 
-template<ConstexprString type_iri_t>
-struct LiteralDatatypeImpl {
-private:
-    /**
-     * static_assert would always trigger if it wasn't dependent on a template parameter.
-     * With this helper template, it only triggers if the function is instantiated.
-     */
-    template<typename>
-    using always_false = std::false_type;
-    template<typename T>
-    static constexpr bool always_false_v = always_false<T>::value;
+namespace capabilities {
```

Comment:
I thought about it but I don't see what value that adds, since typedefs are not "inherited" by default, so
you have to put in a using decl anyways (if that was your intention).

Reply:
yes, I forgot about the thing with the using declarations. I agree.